src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-generate.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Generate QRCode for Charging Station",
+  "type": "object",
+  "properties": {
+    "ChargingStationID": {
+      "$ref": "chargingstation.json#/definitions/id"
+    },
+    "ConnectorID": {
+      "$ref": "chargingstation.json#/definitions/connectorID"
+    },
+    "ProjectFields": {
```

Comment:
I think there is no projected fields in downloading the QR-Code as you get a PDF

Reply:
Indeed, removed